airsonic-main/src/main/java/org/airsonic/player/service/AudioScrobblerService.java

```diff
@@ -285,39 +381,66 @@ private RegistrationThread() {
         public void run() {
             while (true) {
                 RegistrationData registrationData = null;
+                int succeedCount = 0;
                 try {
                     registrationData = queue.take();
-                    scrobble(registrationData);
+                } catch (Exception x) {
+                    LOG.error("Failed to deque registration.", x);
+                }
+
+                LOG.info("Begin registration for " + registrationData.title + ".");
```

Comment:
I would suggest all these `registrationData.title` values in log messages be (double-)quoted.